src/FirstSteps/RPG/Heroes/Boss.cs

```diff
@@ -0,0 +1,21 @@
+﻿
+namespace FirstSteps.RPG.Heroes
+{
+    public class Boss : IAttacker
```

Comment:
Is it like constuctor? i have to set all variables to the public to get access. I don't know how can I use that method in other class? what is reason? With that method I still have to create a new object: Boss boss = new Boss(){ name ="diablo) etc.
i tried to do it like this" Boss boss = new Boss.Creatediablo( name = "Diablo"....) could you explain this? I have not yet encountered such a case

Reply:
And i need set boss values in hell class?

Reply:
You don't have to use the `new` keyword because these methods are static and static methods are referenced through the type name. So in this case `var result = Result.CreateSuccess(99);` or `var result = Result.CreateFail()`.

In your example, the `Hell` class has to know everything about the boss called Diablo. When you add static methods in the `Boss` class you will able to write code like this: `Boss diablo = Boss.CreateDiablo();`